core/Cargo.toml

```diff
@@ -15,13 +14,13 @@ path = "lib.rs"
 
 [dependencies]
 downcast-rs = "1.1.1"
-futures = { version = "0.3.4", features = [ "thread-pool", "compat" ] }
+futures = { version = "0.3.4", features = ["thread-pool", "compat"] }
 lazy_static = "1.4.0"
 libc = "0.2.67"
 log = "0.4.8"
 serde_json = "1.0.48"
 url = "2.1.1"
-rusty_v8 = "0.3.5"
+rusty_v8 = { version = "0.3.7", features = ["binary"] }
```

Comment:
It should be optional - homebrew needs to build from source

Reply:
How could we make it optional?

Reply:
It’s why I was thinking about an env var.

But maybe adding a binary feature to deno_core to turn on the rusty v8 feature?